www/Themes/Centreon-2/style.css

```diff
@@ -1414,7 +1414,7 @@ table.ListTableSmallArg td {
 .ListTable,.ListTableMedium,.ListTableSmall,.ListTableMultiTp {border-collapse:separate;border-spacing:0;border-style:solid;empty-cells:show;}
 .ListTable,.ListTableMedium,.ListTableSmall {border-width:1px;}
 .ListTable td,.ListTableMedium td,.ListTableSmall td,.ListTableSmallArg {padding-bottom:2px;padding-top:2px;}
-.ListTable td {padding: 4px; overflow: hidden; text-overflow: ellipsis;}
+.ListTable td {padding: 4px;}
```

Comment:
be carefull that it does not break other listing pages

Reply:
This was added by Edyta and NIkola, so that's an revert